src/main/java/org/dependencytrack/persistence/ProjectQueryManager.java

```diff
@@ -965,7 +965,6 @@ public void recursivelyDelete(final Project project, final boolean commitIndex)
         delete(project.getMetadata());
         delete(project.getProperties());
         delete(getAllBoms(project));
-        delete(project.getChildren());
```

Comment:
Why was this removed?

Reply:
Because, at some point, I got an exception here, since the children are already deleted in lines 941 to 945.